pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+addopts = --color=yes
```

Comment:
With this configuration, `pytest` is equivalent to `pytest --color=yes`.

https://docs.pytest.org/en/latest/customize.html#adding-default-options

Reply:
It seems all the pytest commands contain `--verbose`. Should it also be included in `addopts`?

Reply:
I guess the small test runner doesn't have `--verbose`, so can probably just keep this here.